NoteFind/View Controllers/FeedViewController.h

```diff
@@ -12,11 +12,14 @@
 #import "LoginViewController.h"
 #import "Note.h"
 #import "NoteCell.h"
+#import <CoreData/CoreData.h>
```

Comment:
Forward declare `NSPersistentContainer`

Reply:
i meant to take this out i don't use the container so i'll delete that line & the container